ocaml/testsuite/tests/typing-missing-cmi/test.ml

```diff
@@ -19,5 +19,3 @@ module = "main.ml"
 ocamlc_byte_exit_status = "2"
 ******** check-ocamlc.byte-output
 *)
-
-(* CR layouts v2.9: error message here is unreviewed *)
```

Comment:
`test.compilers.reference` says:

```
File "main.ml", line 1, characters 8-11:
1 | let _ = A.a = B.b
            ^^^
Error: This expression has type M.a but an expression was expected of type
         ('a : value)
       The layout of M.a is any, because
         the .cmi file for M.a is missing.
       But the layout of M.a must be a sublayout of value, because
         of layout requirements from an imported definition.
```
I don't know what "layout requirements from an imported definition" means. Is it possible to be more specific as to which definition? (Or which kind of definition?) Otherwise, I find this to be more confusing than helpful.

Reply:
Also, I think this file is stale and is not checked by the test suite:

```
ocaml/testsuite/tests/typing-missing-cmi/main.ml.reference
```

Reply:
Stale file removed. Filed ticket for the `of layout requirements from an imported definition` problem. There's a fix to have better messages for type parameters of imported types but not yet for imported functions.